runtime/src/accounts_index.rs

```diff
@@ -85,6 +85,12 @@ impl<T> AccountMapEntryInner<T> {
     }
 }
 
+pub enum AccountIndexGetResult<T: 'static> {
+    Success(ReadAccountMapEntry<T>, usize),
+    NoRootFound(ReadAccountMapEntry<T>),
```

Comment:
nit: Rename `NoRootFound` to `NotFoundOnFork`, as this also encapsulates not matching non-rooted ancestors

Reply:
done